workflows/execute_workflows.sh

```diff
@@ -42,17 +42,27 @@ save_workspaces() {
     mv "$WORKSPACE_DIR"/"$2".zip "$WORKFLOW_DIR"/results/"$2"_"$WORKFLOW_NAME".zip
 }
 
-mkdir "$WORKSPACE_DIR"
-mkdir -p "$WORKFLOW_DIR/nf-results"
+source "$ROOT"/venv/bin/activate
 
 echo "Initialize submodules …"
 git submodule update --init --recursive
 
 # update the data from quiver-data repository if necessary
 echo "Update quiver-data if necessary …"
-cd submodules/quiver-data || exit
-git fetch
-git merge origin/main
+git -C submodules/quiver-data pull origin/main
+
+echo "Installing OtoN …"
+pushd submodules/oton
```

Comment:
```suggestion
pushd submodules/oton || exit
```

Reply:
Why would that be the case? Also, `|| { echo "Reason why we exit"; exit 1; }` is a better pattern IMHO. A simple exit (== `exit 0`) implies successful execution.

Reply:
Would you suggest to omit it? (It's something my linter complained about, that's why I added it. But I see your point of `|| exit` not having a point.)

Reply:
The only reason why this might be the case would be that the directory does not exist. Which would mean that something weird happened because we create that directory in the script. So the best mitigation is to let it raise an "no such file or directory" error.

So yeah, don't do `|| exit`.

What linter proposes that? shellcheck?

Reply:
Yep, shellcheck. Do you have a recommendation for a better linter?

Reply:
No, shellcheck is a great linter and the reasoning behind https://www.shellcheck.net/wiki/SC2164 is sound but I think it can be disabled because we already `set -e`, which will exit the script with non-zero status in case `cd/pushd` fails.

Reply:
Alrighty.